orchestrators/ProcurementOperations/src/Services/EarlyPaymentDiscountService.php

```
@@ -38,45 +42,47 @@ public function __construct(
 
     /**
      * {@inheritDoc}
      */
     public function calculateEarlyPaymentDiscountAmount(
-        Money $invoiceAmount,
         EarlyPaymentDiscountData $discountTerms,
+        Money $invoiceAmount,
         \DateTimeImmutable $paymentDate,
     ): Money {
         if (!$discountTerms->isDiscountAvailable($paymentDate)) {
             return Money::of(0, $invoiceAmount->getCurrency());
```

PHPMD (2.15.0)

[error] Avoid using static access to class '\Nexus\Common\ValueObjects\Money' in method 'calculateEarlyPaymentDiscountAmount'. (undefined)

(StaticAccess)